### PR TITLE
Add contributor instructions to root path

### PR DESCRIPTION
**Pull request by ajclose**
#54 Created new root path and added instructions from google doc.

### CODE REVIEW

**Comment by jwo** [comment]
@ajclose I know this is scope creep.... But could we move the splash page into a model?

We're going live with Irma-api, and I think being able to edit the splash page content could be very useful.

If you'd like to merge _this_ and then do the content in a separate PR we can do that too

File: db/migrate/20170905201043_create_pages.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class CreatePages < ActiveRecord::Migration[5.1]
+  def change
+    create_table :pages do |t|
+      t.string :key, unique: true, null: false
```
Comment (cmason) [inline db/migrate/20170905201043_create_pages.rb]: I think the index must be specified like: 
```
      t.string :key, index: { unique: true }, null: false
```
in order to generate a unique index.

File: test/fixtures/pages.yml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  key: MyString
+  content: MyText
+
+two:
+  key: MyString
```
Comment (cmason) [inline test/fixtures/pages.yml]: It shouldn't be possible to create multiple pages with the same key.

**Review by jwo — APPROVED** [review]
👍 Thanks @ajclose for your first contribution!

**Pull request merged**